utils/navigator_robotx_comms/test/robotx_comms.test

```diff
@@ -0,0 +1,8 @@
+<launch>
+    <test test-name="robotx_comms_test" pkg="navigator_robotx_comms" type="robotx_comms_test.py" />
+</launch>
+
+	<test test-name="set_localhost_params" pkg="navigator_robotx_comms" type="set_localhost_params.py" />
```

Comment:
This is still not needed, just set the params in the launch file, like:
```
 <node name="navigator_robotx_comms" pkg="navigator_robotx_comms" type="navigator_robotx_comms.py" >
   <param name="~my_param" value="foo" />
</node>
```

Reply:
Note that the `~` makes it a "private" parameter, so in the node it should be accessed as `rospy.get_param("~my_param")